File: services/gui/app/controllers/landingPage.js
const landingPage = (req, res) => {
    if(req.get('Accept') === "application/json"){
        res.json({message: "Welcome to " + process.env.RELEASE + "!"});
        return;
    }
    res.render("landingPage/index");
};

module.exports = {
    landingPage
}

File: services/gui/app/controllers/landingPage.go
package controllers

import (
	"net/http"
)

func LandingPage(w http.ResponseWriter, r *http.Request) {
	if r.Header.Get("Accept") == "application/json" {
		w.Header().Set("Content-Type", "application/json")
		w.Write([]byte(`{"message": "Welcome to ` + os.Getenv("RELEASE") + `!"}`))
		return
	}
	http.ServeFile(w, r, "views/landingPage/index.html")
}
